server/routes/questions/addEditor.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express')
+const { verifyUserAccessToken } = require('../middlewares')
+const router = express.Router()
+const ajv = require('../../schema')
+const { addEditorSchema } = require('../../schema/questions')
+const { addEditor } = require('../../models/questions')
```

Comment:
make it `addOrRemoveEditorSchema`...more convenient if using one file for both

Reply:
It's fine, any ways